src/api/utils/helpers.py

```diff
@@ -199,7 +199,9 @@ def response_builder(data, status_code=200):
     return response
 
 
-def add_extra_user_info(token, user_id, url=os.environ.get('ANDELA_API_URL')):  # pragma: no cover
+def add_extra_user_info(
+    token, user_id, url=os.environ.get('ANDELA_API_URL')
+):  # pragma: no cover
```

Comment:
The goal was to exclude this from testing, does this still work?

Reply:
Yes, it does.